src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -287,7 +289,7 @@ export default class UploadMemorandum extends Mixins(CommonMixin, DocumentMixin)
     this.uploadMemorandumDocKey = null
   }
 
-  private isFileUploadValidFn (val) {
+  private async isFileUploadValidFn (val) {
```

Comment:
This doesn't need to be async, since nothing is awaited inside this method.

Reply:
removed `async`